nuget.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Love it.

Reply:
This is my favorite. No idea, what's going on. Maybe encoding stuff.